src/main/java/io/github/syst3ms/skriptparser/expressions/CondExprEmpty.java

```diff
@@ -0,0 +1,60 @@
+package io.github.syst3ms.skriptparser.expressions;
+
+import java.util.Arrays;
+
+import org.jetbrains.annotations.Nullable;
+
+import io.github.syst3ms.skriptparser.Parser;
+import io.github.syst3ms.skriptparser.lang.Expression;
+import io.github.syst3ms.skriptparser.lang.TriggerContext;
+import io.github.syst3ms.skriptparser.lang.base.ConditionalExpression;
+import io.github.syst3ms.skriptparser.parsing.ParseContext;
+
+/**
+ * Check if a given string or list is empty.
+ *
+ * @name Empty
+ * @pattern %strings/objects% (is|are)[1:( not|n't)] empty
+ * @since ALPHA
+ * @author Olyno
+ */
+public class CondExprEmpty extends ConditionalExpression {
+
+    static {
+        Parser.getMainRegistration().addExpression(CondExprEmpty.class,
+            Boolean.class,
+            true,
+            2,
+            "%strings/objects% (is|are)[1:( not|n't)] empty"
```

Comment:
An issue I see with this syntax is that if Mr. Skript Skriptington over here wants to check if their list of strings is empty (the list, not the strings themselves), but his list is `["", " ", "   ", ""]`, then this condition will return true, and that's not the desired behavior. 

This is indeed an edge case, but just for the sake of not having to rely on some horrible hack, there should be a way to force the syntax to use one given interpretation. Something like `if {list::*} is an empty list` would work for example.